alerter/AlerterTest.java

```diff
@@ -0,0 +1,9 @@
+
+public class AlerterTest {
+	public static void testAlertFailureCount() {
+		alerter.alertInCelcius(500.0f);
+		assert (alerter.alertFailureCount == 1);
+		alerter.alertInCelcius(450.0f);
+		assert (alerter.alertFailureCount == 2);
```

Comment:
There should be one test case for lower than threshold value and on for == threshold. This clearly defines what the production code should output in various cases.

Reply:
Resolved, Could you please validate now. Thankyou!!